scripts/daqconf_viewer

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
+import asyncio
+import copy
+import httpx
+import json
+import sys
+
+from rich.text import Text
+from difflib import context_diff, ndiff, unified_diff
+
+from textual import log, events
+from textual.app import App, ComposeResult
+from textual.containers import Horizontal, Content, Container, Vertical
+from textual.widget import Widget
+from textual.widgets import Button, Header, Footer, Static, Input, Label, ListView, ListItem, Tree
+from textual.reactive import reactive, Reactive
+from textual.message import Message, MessageTarget
+from textual.screen import Screen
+
+auth = ("fooUsr", "barPass")
+oldconf = None
+
+class LabelItem(ListItem):
+    def __init__(self, label: str) -> None:
+        super().__init__()
+        self.label = label
+
+    def compose( self ) -> ComposeResult:
+        yield Label(self.label)
+
+class Configs(Static):
+    conflist = reactive([])
+
+    def __init__(self, hostname, **kwargs):
+        super().__init__(**kwargs)
+        self.hostname = hostname
+
+    def on_mount(self) -> None:
+        self.set_interval(0.1, self.update_configs)
+
+    def compose(self) -> ComposeResult:
+       yield ListView(LabelItem("asfas"))
+
+    async def update_configs(self) -> None:
+        async with httpx.AsyncClient() as client:
+            r = await client.get(f'{self.hostname}/listConfigs', auth=auth, timeout=60)
```

Comment:
and here too

Reply:
Since this is a different widget, I did this by having the except block call a function that finds the display and puts the appropriate text there.